app/api/users/index.js

```diff
@@ -126,6 +127,23 @@ router.get('/subcheck/:sub', async (request, response) => {
     });
 });
 
+router.get('/:id/submissions', async (req, res) => {
+  const userId = req.params.id;
+  try {
+    const campaignsAndSubmissions = await ApplicationSubmission.findCampaignsByUserSubmissions(userId);
+    if (campaignsAndSubmissions) {
+      res.status(200).json({ campaignsAndSubmissions, error: null });
```

Comment:
This makes it sound like there are two types of data in one array, when it's actually just submissions, right? Can we rename this var to just `submissions`?